src/WSRequestHandler_SceneCollections.cpp

```diff
@@ -12,26 +17,32 @@
  * @category scene collections
  * @since 4.0.0
  */
-RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest& request) {
-	if (!request.hasField("sc-name")) {
+RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest &request)
+{
+	if (!request.hasField("sc-name"))
+	{
 		return request.failed("missing request parameters");
 	}
 
-	const char* sceneCollection = obs_data_get_string(request.parameters(), "sc-name");
-	if (!sceneCollection) {
+	const char *sceneCollection = obs_data_get_string(request.parameters(), "sc-name");
```

Comment:
Same as with references:
```suggestion
	const char* sceneCollection = obs_data_get_string(request.parameters(), "sc-name");
```